src/plugins/phoenix.js

```diff
@@ -22,6 +22,41 @@ export default {
         },
         // FIXME: optional publicContext parameter is a mess
         downloadFile(file, publicContext = null) {
+          if (publicContext === null) {
+            const url = this.$client.helpers._webdavUrl + file.path
+
+            return this.$client.signUrl(url).then(signedUrl => {
+              fetch(signedUrl, {
```

Comment:
in any case this should be  `fetch(url, {`

Reply:
implemented it that way. We need to keep the HEAD request for now, to show errors in the web ui instead of seeing them in the download progress list of the browser.